ansible/roles/ansible-horizon-plugin/tasks/collect_compress.yml

```diff
@@ -43,7 +43,25 @@
   shell: echo "HORIZON_CONFIG['customization_module'] = 'dashboards.overrides'" > `echo {{os_local_settings_path.stdout}} | cut -d ' ' -f1 |{ read trilio_setting_path ; echo $trilio_setting_path/_001_trilio_dashboard.py ; }`
   when:
     - os_local_settings_path != ""
-    
+
+- name: Enable OPENSTACK_ENCRYPTION_SUPPORT
+  lineinfile:
+    path: "{{os_local_settings_path.stdout}}/_001_trilio_dashboard.py"
+    regexp: 'OPENSTACK_ENCRYPTION_SUPPORT = False'
+    line: OPENSTACK_ENCRYPTION_SUPPORT = True
+  when:
+   - ((groups['key-manager_hosts'] is defined) and (groups['key-manager_hosts'] | length > 0))
+   - os_local_settings_path != ""
+
+- name: Enable TRILIO_ENCRYPTION_SUPPORT
+  lineinfile:
+    path: "{{os_local_settings_path.stdout}}/_001_trilio_dashboard.py"
+    regexp: 'TRILIO_ENCRYPTION_SUPPORT = False'
```

Comment:
no need to check for the entire line, look for the TRILIO_ENCRYPTION_SUPPORT, if present then replace the line.
Else, add a new line